DOCS-8381 SAP SuccessFactors 3.2 (#875)
mulesoft/docs-connectors: sap-successfactors/3.2/modules/ROOT/pages/sap-successfactors-connector-reference.adoc

= SAP SuccessFactors Connector 3.2 Reference - Mule 4
:page-aliases: connectors::sap/sap-successfactors-connector-reference.adoc

Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]

Anypoint Connector for SAP SuccessFactors (SuccessFactors Connector) provides full support to query, create, update, and delete entities using the OData API exposed by SuccessFactors.

Release Notes: xref:release-notes::connector/sap-successfactors-connector-release-notes-mule-4.adoc[SAP SuccessFactors Connector Release Notes]

[[config]]
== Default Configuration

=== Parameters

[%header%autowidth.spread]
|===
| Name | Type | Description | Default Value | Required
|Name | String | The name for this configuration. Connectors reference the configuration with this name. | |x
| Connection a| <<config_basic, Basic Authentication>>

>  | The connection types that can be provided to this configuration. | |x

Reviewer: ```suggestion
 | The connection types to provide to this configuration. | |x
```

> | Time Zone a| String |  Date objects returned from SuccessFactors will be converted to the specified timezone. |  `UTC` | 

Reviewer: ```suggestion
| Time Zone a| String |  Date objects returned from SuccessFactors are converted to the specified timezone. |  `UTC` | 
```
